File: jamZirBaze.java
import java.util.Scanner;

public class jamZirBaze {
    static Scanner input = new Scanner(System.in);
    // static Scanner input2 = new Scanner(System.in);
    public static void main(String[] args) {

        int sum = 0 ;

        int n = input.nextInt();
        int[] li = new int[n];

        for (int i=0 ; i<n ; i++)
        {
            int an = input.nextInt();
            li[]
        }
        // String[] li = an.split(" ");

        for (int i=0 ; i<n/2 ; i++)
        {
            int n2 = n -1 -i ;
            int tedad = count(i , n);
            int x1 = Integer.parseInt(li[i]);
            int x2 = Integer.parseInt(li[n2]);
            sum += tedad*(x1+x2);
        }

        if (isOdd(n))
        {
            int n2 = (n/2);
            int tedad = count(n2 , n);
            sum += tedad*(Integer.parseInt(li[n2]));
        }
        
        System.out.println(sum);
    }

    static boolean isOdd(int num)
    {
        if (num%2 == 0)
        {
            return false;
        }else
        {
            return true;
        }
    }

    static int count(int i , int n)
    {
        return (n-i)*(i+1);
    }
}

File: jamZirBaze.py
n = int(input())
an = input()
jam = 0

an = (an.split(" "))

def isOdd(n):
    if n%2 == 0:
        return False
    else:
        return True

def count(i , n):
    return (n-i)*(i+1)

for i in range(n//2):
    n2 = n -1 -i
    tedad = count(i , n)
    # print(f"{i+1}&{n2+1} - {an[i]}&{an[n2]} - {tedad}")
    jam += tedad*(int(an[i])+int(an[n2]))
    # print(jam)

if isOdd(n):
    n2 = (n//2)
    tedad = count(n2 , n)
    # print(f"{n2} - {an[n2+1]} - {tedad}")
    jam = jam + tedad*int(an[n2])
    # print(jam)

print(jam)